stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/extensions/internal/Channel.kt

```diff
@@ -192,7 +193,22 @@ internal fun Channel.incrementUnreadCount(currentUserId: String, lastMessageSeen
 }
 
 internal fun Collection<Channel>.applyPagination(pagination: AnyChannelPaginationRequest): List<Channel> {
-    return asSequence().sortedWith(pagination.sort.comparator)
+    val logger = StreamLog.getLogger("ChannelSort")
+
+    return asSequence()
+        .also { channelSequence ->
+            if (BuildConfig.DEBUG) {
+                val ids = channelSequence.joinToString { channel -> channel.id }
+                logger.d { "Sorting channels: $ids" }
+            }
```

Comment:
You don't need to check `DEBUG` flag here, cause it happens internally in the `StreamLog` inline function. So we can simplify this to:
```suggestion
           logger.d {
               "Sorting channels: ${channelSequence.joinToString { channel -> channel.id }}"
           }
```